apps/ledger-live-desktop/src/renderer/analytics/segment.ts

```diff
@@ -115,19 +115,34 @@ const getPtxAttributes = () => {
       ? stakingProviders?.params?.listProvider.length
       : "flag not loaded";
 
-  const stakingCurrenciesEnabled =
-    stakePrograms?.enabled && stakePrograms?.params?.list?.length ? stakePrograms.params.list : [];
-  const partnerStakingCurrenciesEnabled =
+  const stakingCurrenciesEnabled: string[] | string =
+    stakePrograms?.enabled && stakePrograms?.params?.list?.length
+      ? stakePrograms.params.list
+      : "flag not loaded";
+  const partnerStakingCurrenciesEnabled: string[] | string =
     stakePrograms?.enabled && stakePrograms?.params?.redirects
       ? Object.keys(stakePrograms.params.redirects)
-      : [];
+      : "flag not loaded";
+
+  /** Tether USDT provider is proxy for stablecoin flow rollout.  */
+  const usdtProvider =
+    !stakePrograms?.enabled || !stakePrograms?.params?.redirects
+      ? undefined
+      : stakePrograms?.params?.redirects["ethereum/erc20/usd_tether__erc20_"]?.platform;
```

Comment:
```suggestion
  const usdtProvider = stakePrograms?.enabled && stakePrograms?.params?.redirects
  ? stakePrograms.params.redirects["ethereum/erc20/usd_tether__erc20_"]?.platform
  : undefined;
```

I think it's more readable and easy to understand to not use the "negative" 8 assertion here